Lightyear/Core.idr

```diff
@@ -153,4 +153,13 @@ satisfy : (Monad m, Stream tok str)
                    => (tok -> Bool)
                    -> ParserT str m tok
 satisfy p = satisfyMaybe (\t => if p t then Just t else Nothing)
+
+||| Succeed if and only if the argument parser fails.
+notFollowedBy : ParserT str m tok -> ParserT str m ()
```

Comment:
I actually find this name a bit strange because it sounds like a binary operation: <code>foo &#96;notFollowedBy&#96; bar</code>. Perhaps a name like `forbid`, `failOn` or `expectFailure` would be better.

What do you think?